### ISSUE TITLE
<img> closing tags

### ISSUE THREAD

**Issue by nikhilm19** [mertJF/tailblocks#45]
It would be great if the code snippets had closing tags to <img>

JSX doesn't allow unclosed <img> tag!

**Comment by MrGKanev** [comment]
You can read [MDN web doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Specifications) and understand in HTML 5 its valid without the "closed tag". When using react a lot of tags need to be "closed". Not everybody works with it so I'm not sure if this is a good idea.

**Comment by rolandleth** [comment]
Think of it like this, though: for React `>` is invalid and `/>` is needed; for everyone else `/>` is not needed, but it's not invalid either, it's still valid HTML 5 syntax. Adding it would help React users, while not hurting anyone else.

**Comment by nikhilm19** [comment]
> Think of it like this, though: for React `>` is invalid and `/>` is needed; for everyone else `/>` is not needed, but it's not invalid either, it's still valid HTML 5 syntax. Adding it would help React users, while not hurting anyone else.

That was my point. If it works for some without breaking for others, it can be included.

**Comment by ManasMadrecha** [comment]
Yes, closing tags should be there. They are needed even in markdown files of Nuxt Content. Thanks for this amazing repo. ❤

**Comment by julbrs** [comment]
Got the same issue for `<input>` and `<br>`. I am adding a 👍   for closed tags too, thanks for this nice work by the way !
